src/modules/users/dto/wallet-callback.dto.ts

```
@@ -0,0 +1,36 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString, IsNotEmpty } from 'class-validator';
+
+export class WalletCallbackDto {
+  @ApiProperty({
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)